### PR TITLE
Use http-client fork again

### PR DESCRIPTION
**Pull request by akshaymankar**
The fork has been updated with latest http-client. The new logic we wrote for verifying the pinned public key fingerprints (for bots and legalhold) is wrong, so we go back to old ways of verifying these fingerprints.

## Checklist

 - [x] Add a new entry in an appropriate subdirectory of `changelog.d` - This is majorly a revert so the changelogs also get reverted.
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

File: nix/haskell-pins.nix
Comment (MangoIV) [inline nix/haskell-pins.nix:131]: should I open a PR against our master so this is visible better?

**Review by MangoIV — APPROVED** [review]
Looks good to me.

File: nix/haskell-pins.nix
Comment (akshaymankar) [reply nix/haskell-pins.nix:131]: Its actually here: https://github.com/wireapp/http-client/pull/5
I am waiting for the CI to be green and us to verify that bots work again. Once that happens we can merge that PR and update the revision here.

**Comment by akshaymankar** [comment]
Didn't wait for CI because the last commit didn't even change the sha256 of the http-client fork.

**Pull request merged**
